lib/node_modules/@stdlib/_tools/scaffold/math-iter-unary/scripts/scaffold.sh

```diff
@@ -104,7 +104,7 @@ camelcase_to_snakecase() {
 }
 
 # Convert the alias to snakecase:
-alias_snakecase=$(camelcase_to_snakecase "${alias:4}")
+alias_snakecase=$(camelcase_to_snakecase "${alias}")
```

Comment:
It's not clear why you'd need to change this or remove the unary function alias logic below.

Reply:
Understood this now, I've reverted this change.